release/models/policy-forwarding/openconfig-pf-forwarding-policies.yang

```diff
@@ -253,12 +260,9 @@ submodule openconfig-pf-forwarding-policies {
 
     leaf network-instance {
       type leafref {
-
-        // We are at:
-        // $NIROOT/policy-forwarding/policies/
-        // policy/rules/rule/action/config/
-        // network-instance
-        path "../../../../../../../../config/name";
+        // this must be an absolute reference to allow another NHG to be
```

Comment:
Rob - just a nit. In the comments, did we mean "another NI" instead of "another NHG"?

Reply:
Yes, apologies. Fixed.